tests/common.js

```diff
@@ -1,61 +1,100 @@
+'use strict';
 const chai = require('chai'), expect = chai.expect;
 chai.use(require('chai-string'));
 
 module.exports = {
+
   login: function(browser, username, password, domain) {
-    return browser.url('/').getTitle()
-      .then(title => expect(title).to.startWith('hiveIO Appliance |'))
-      .then(() => browser.setValue('//input[@name="username"]', username))
-      .then(() => browser.setValue('//input[@name="password"]', password))
-      .then(() => browser.selectByVisibleText('//select[@name="domain"]', domain))
+    return browser.url('/')
+      .then(() => browser.waitForExist('//div[@class="input-group margin-bottom-sm"]//input[@name="username"]', 10000))
+      .then(() => browser.setValue('//div[@class="input-group margin-bottom-sm"]//input[@name="username"]', username))
+      .then(() => browser.waitForExist('//div[@class="input-group"]//input[@name="password"]', 10000))
+      .then(() => browser.setValue('//div[@class="input-group"]//input[@name="password"]', password))
+      .then(() => browser.waitForExist('//div[@class="input-group"]//select[@name="domain"]'), 10000)
+      .then(() => browser.selectByVisibleText('//div[@class="input-group"]//select[@name="domain"]', domain))
+      .then(() => browser.waitForExist('//button[@type="submit"]'), 10000)
       .then(() => browser.click('//button[@type="submit"]'))
-
-      // wait for the page to show up, default page is the "Overview"
-      .then(() => browser.waitForExist('h1.page-header'))
-      .then(() => browser.getText('h1.page-header'))
-      .then(header => expect(header).to.equal('Overview'));
+      .then(() => browser.waitForExist('//h1[@class="page-header"]'))
+      .then(() => browser.waitUntil(() => 
+        browser.getText('//h1[@class="page-header"]')
+        .then((text) => { return text === 'Overview'; })
```

Comment:
Not sure I like the success criteria - and if we do use it, I suggest we set it as a variable, that defaults to 'Overview' - if we want to use the same common for testing of other sites, like the broker, the landing page will be different...

alternatively, look for different success criteria (session cookie? JWT down the road, etc.(